src/azul/openapi.py

```diff
@@ -0,0 +1,113 @@
+import copy
+from typing import MutableMapping, List
+
+from azul.deployment import aws
+from azul.types import JSON
+
+
+def spec(description: JSON):
```

Comment:
In the `@app.route(…, openapi=…` approach, our `AzulChalicApp.route` override would call this function. The reason I like tying this to `app.route` is that `@spec` doesn't make sense by itself, it should only be used on routed functions.

Reply:
I see. OK I'll give it a whirl.